src/main/java/com/google/devtools/build/lib/skyframe/RepoFileFunction.java

```diff
@@ -131,7 +126,7 @@ private static StarlarkFile readAndParseRepoFile(Path path, Environment env)
     return starlarkFile;
   }
 
-  private static String getDisplayNameForRepo(
+  public static String getDisplayNameForRepo(
```

Comment:
IMO we should just get rid of this method now that the `mainRepoMapping` arg is always null (in this class). The current caller should just use `repoName.toString()`. Then we can move this method to its only remaining call site.

Reply:
If you don't mind, I won't do this in this change; it looks like a subtle behavior change that might make a difference and thus it makes more sense in a separate change.

Reply:
SGTM!